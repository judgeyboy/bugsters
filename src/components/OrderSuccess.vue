<template>
  <div class="order-success text-center">
    <img :src="require('@/assets/cute-bug.png')" width="250">
    <h1 style="color: #444BCE; font-size: 3.5em; font">Good News!</h1>
    <p>
      Great! We've received your request.
      We'll plan the test and send you an offer within two working days.
      Your request number is <strong>{{ orderId }}</strong>.
    </p>
    <p>
      In the mean time, if you have any questions, please don't hesitate to
      <scroll-link location="#contact">contact us</scroll-link>
    </p>
  </div>
</template>

<script>
import ScrollLink from '@/components/ScrollLink'

export default {
  name: 'OrderSuccess',
  props: ['orderId'],
  components: {
    ScrollLink
  }
}
</script>

<style lang="sass" scoped>
h1
  color: #444BCE
  font-size: 3.5em
  font-weight: 500

p
  font-size: 1.5em
  font-weight: 500

</style>
