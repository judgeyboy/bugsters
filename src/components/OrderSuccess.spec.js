import { shallow } from 'vue-test-utils'

import OrderSuccess from './OrderSuccess'

describe('OrderSuccess Component', () => {

  it('should be a Vue instance', () => {
    const wrapper = shallow(OrderSuccess)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(OrderSuccess)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
