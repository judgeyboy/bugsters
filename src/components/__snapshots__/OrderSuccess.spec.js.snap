// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderSuccess Component should match snapshot 1`] = `
"<div class=\\"order-success\\"><h2 class=\\"text-center\\">Success :)</h2> <p>
    Great! We've received your request.
    We'll plan the test and send you an offer within two working days.
    Your request number is <strong>#20948</strong>.
  </p> <p>
    In the mean time, if you have any questions, please don't hesitate to contact us
  </p></div>"
`;
